docs/index.md

```diff
@@ -274,3 +205,35 @@ resides in. Alternatively you can provide this value as an environment variable
 Where there are multiple environment variable options, the `DATABRICKS_AZURE_*` environment variables takes precedence 
 and the `ARM_*` environment variables provide a way to share authentication configuration when using the `databricks-terraform` 
 provider alongside the `azurerm` provider.
+
+# Environment variables
+
+The following variables can be passed via environment variables:
+
+* `host` → `DATABRICKS_HOST`
+* `token` → `DATABRICKS_TOKEN`
+* `basic_auth.username` → `DATABRICKS_USERNAME`
+* `basic_auth.password` → `DATABRICKS_PASSWORD`
+* `config_file` → `DATABRICKS_CONFIG_FILE`
+* `managed_resource_group` → `DATABRICKS_AZURE_MANAGED_RESOURCE_GROUP`
+* `azure_region` → `AZURE_REGION`
+* `workspace_name` → `DATABRICKS_AZURE_WORKSPACE_NAME`
+* `resource_group` → `DATABRICKS_AZURE_RESOURCE_GROUP`
+* `subscription_id` → `DATABRICKS_AZURE_SUBSCRIPTION_ID` or `ARM_SUBSCRIPTION_ID`
+* `client_secret` → `DATABRICKS_AZURE_CLIENT_SECRET` or `ARM_CLIENT_SECRET`
+* `client_id` → `DATABRICKS_AZURE_CLIENT_ID` or `ARM_CLIENT_ID`
+* `tenant_id` → `DATABRICKS_AZURE_TENANT_ID` or `ARM_TENANT_ID`
+
+For example you can have the following provider definition:
+
+``` hcl 
+provider "databricks" {}
+```
+
+Then run the following code and the following environment variables will be injected into the provider.    
+
+``` bash
+$ export HOST="http://databricks.domain.com"
+$ export TOKEN="dapitokenhere"
+$ terraform plan
```

Comment:
Changing these to DATABRICKS_HOST, DATABRICKS_TOKEN

Reply:
@stikkireddy good catch!